src/components/ChatLog.js

```diff
@@ -0,0 +1,26 @@
+import React from 'react';
+import ChatEntry from './ChatEntry';
+import './ChatLog.css';
+import PropTypes from 'prop-types';
+
+
+const ChatLog = ({entries,updateLikes}) => {
+
+    return (
+    <div className="chat-log">
```

Comment:
You forgot to add your `key` prop to your sibling components. Remember, when you make a list of components with the `map` method React will give you a warning saying that each component needs a unique `key`. You can fix that warning with this here: 

```javascript
 {entries.map((entry, index) => (
            <ChatEntry 
            sender={entry.sender}
            key{index}
            body = {entry.body}
            timeStamp = {entry.timeStamp} 
            updateLikes ={updateLikes}
            />
        ))}
```
Here we used the `index` of each entry to give them a unique `key`. Usually this would be an id associated with the object being used, but for now this will suffice.